deployment/chatwoot_nginx.conf

```diff
@@ -0,0 +1,47 @@
+server {
```

Comment:
we should probably rename this file to `nginx_chatwoot.conf` since that's the name used in the script.

Reply:
Down, sorry for bad name ;)